@datapythonista I got the `:py:func:` syntax from http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py:func.

I tried building the docs but got the following error message:
```console
$ ./make.py html
Running Sphinx v1.7.5

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File "/Users/Adam/pandas/pandas/__init__.py", line 26, in <module>
    from pandas._libs import (hashtable as _hashtable,
  File "/Users/Adam/pandas/pandas/_libs/__init__.py", line 4, in <module>
    from .tslibs import (
  File "/Users/Adam/pandas/pandas/_libs/tslibs/__init__.py", line 4, in <module>
    from .conversion import normalize_date, localize_pydatetime, tz_convert_single
ModuleNotFoundError: No module named 'pandas._libs.tslibs.conversion'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/Adam/anaconda3/lib/python3.6/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File "/Users/Adam/anaconda3/lib/python3.6/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 140, in <module>
    import pandas
  File "/Users/Adam/pandas/pandas/__init__.py", line 35, in <module>
    "the C extensions first.".format(module))
ImportError: C extension: No module named 'pandas._libs.tslibs.conversion' not built. If you want to import pandas from the source directory, you may need to run 'python setup.py build_ext --inplace --force' to build the C extensions first.
```
What am I missing here?

--- a/pandas/core/generic.py
+++ b/pandas/core/generic.py
@@ -6098,7 +6098,7 @@ class NDFrame(PandasObject, SelectionMixin):
             * 'index', 'values': use the actual numerical values of the index
             * 'nearest', 'zero', 'slinear', 'quadratic', 'cubic',
               'barycentric', 'polynomial' is passed to
-              :func:`scipy.interpolate.interp1d`. Both 'polynomial' and
+              :class:`scipy.interpolate.interp1d`. Both 'polynomial' and
               'spline' require that you also specify an `order` (int),
               e.g. ``df.interpolate(method='polynomial', order=4)``.
               These use the actual numerical values of the index.
@@ -6111,7 +6111,7 @@ class NDFrame(PandasObject, SelectionMixin):
               and `tutorial documentation
               <http://docs.scipy.org/doc/scipy/reference/tutorial/interpolate.html>`__
             * 'from_derivatives' refers to
-              :func:`scipy.interpolate.BPoly.from_derivatives` which
+              :meth:`scipy.interpolate.BPoly.from_derivatives` which
               replaces 'piecewise_polynomial' interpolation method in
               scipy 0.18
 

--- a/pandas/core/generic.py
+++ b/pandas/core/generic.py
@@ -6098,7 +6098,7 @@ class NDFrame(PandasObject, SelectionMixin):
             * 'index', 'values': use the actual numerical values of the index
             * 'nearest', 'zero', 'slinear', 'quadratic', 'cubic',
               'barycentric', 'polynomial' is passed to
-              :py:func:`scipy.interpolate.interp1d`. Both 'polynomial' and
+              :func:`scipy.interpolate.interp1d`. Both 'polynomial' and
               'spline' require that you also specify an `order` (int),
               e.g. ``df.interpolate(method='polynomial', order=4)``.
               These use the actual numerical values of the index.
@@ -6111,7 +6111,7 @@ class NDFrame(PandasObject, SelectionMixin):
               and `tutorial documentation
               <http://docs.scipy.org/doc/scipy/reference/tutorial/interpolate.html>`__
             * 'from_derivatives' refers to
-              :py:func:`scipy.interpolate.BPoly.from_derivatives` which
+              :func:`scipy.interpolate.BPoly.from_derivatives` which
               replaces 'piecewise_polynomial' interpolation method in
               scipy 0.18
 

